### PR TITLE
Improve organization of roles pages

### PR DESCRIPTION
**Pull request by cthoyt**
This PR does the following:

1. Adds link to `roles` page to the navbar
2. Moves the overview of all roles out of the operations committee membership page into its own page
3. Adds links to table of contents

### CODE REVIEW

**Review by nlharris — APPROVED** [review]

File: _includes/docs_toc.md [hunk 1]
```diff
@@ -16,3 +16,7 @@
 - [Ontology tools and resources](/resources)
 
 [Outreach Working Group](/docs/OutreachWG.html)
+
+[Roles](/roles/)
```
Comment (matentzn) [inline _includes/docs_toc.md:20]:
```suggestion
[OBO Foundry Organizational Roles](/roles/)
```

Comment (matentzn) [reply _includes/docs_toc.md:20]: Roles is a bit vague for a heading.

File: _includes/header.html [hunk 2]
```diff
@@ -32,6 +32,7 @@
                             Working Group</a></li>
                         <li><a class="dropdown-item" href="/docs/OutreachWG.html" style="padding-left: 2em">Outreach
                             Working Group</a></li>
+                        <li><a class="dropdown-item" href="/roles/">Roles</a></li>
```
Comment (matentzn) [inline _includes/header.html:35]:
```suggestion
                        <li><a class="dropdown-item" href="/roles/">OBO Foundry Organizational Roles</a></li>
```

**Review by matentzn — APPROVED** [review]
I approve but if you want, feel free to accept my two suggestions which are not that important.

**Comment by matentzn** [comment]
Thank you @cthoyt for this, as always!

**Pull request merged**